lib/spreewald/browser_tab_steps.rb

```diff
@@ -0,0 +1,99 @@
+def browser
+  page.driver.browser
+end
+
+def require_selenium!
+  raise 'This step only works with Selenium' unless selenium_driver?
+end
```

Comment:
I believe those two functions (`browser` and `require_selenium!`) could also be helpful in other steps. How about moving them to a more general place? There are some helpers like this already in `spreewald/lib/spreewald_support`. There's also a file that already is dealing with drivers. What do you think?

Reply:
Thanks for pointing this out, I moved the two helpers to the `DriverInfo` module.